Whiskey/Tasks/Invoke-WhiskeyNUnit3Task.ps1

```diff
@@ -16,7 +16,19 @@ function Invoke-WhiskeyNUnit3Task
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
-    $nunitPackage = 'NUnit.ConsoleRunner'
+    $nunitPackages = @('NUnit.ConsoleRunner')

```

Comment:
Should we just *always* support NUnit2? Or always download the packages but select the runner differently based on the `SupportNUnit2` flag? How big are these packages?

Reply:
I'd be fine with that:

NUnit3 + 1 package, 1.2MB
1.2M    NUnit.ConsoleRunner.3.7.0

NUnit2 + 6 packages, ~500K.
24K     NUnit.Console.3.7.0
48K     NUnit.Extension.NUnitProjectLoader.3.5.0
333K    NUnit.Extension.NUnitV2Driver.3.6.0
52K     NUnit.Extension.NUnitV2ResultWriter.3.5.0
56K     NUnit.Extension.TeamCityEventListener.1.0.2
52K     NUnit.Extension.VSProjectLoader.3.5.0

Reply:
Let's do that then. Always download all the packages but select the "right" one when we need to support NUnit2.